package.json

```diff
@@ -62,7 +62,7 @@
     "next-images": "1.3.0",
     "next-mdx-enhanced": "^2.4.0",
     "next-plugin-yaml": "^1.0.1",
-    "prettier": "^1.19.1",
+    "prettier": "2.0.5",
```

Comment:
Connor's PR contains this original change, but I had to update prettier too so I could push this up